addons/common/CfgFunctions.hpp

```diff
@@ -141,6 +141,12 @@ class CfgFunctions
                 description = "Drops a weapon.";
                 file = "\x\cba\addons\common\fnc_dropWeapon.sqf";
             };
+            // CBA_fnc_execNextFrame
+            class execNextFrame
+            {
```

Comment:
@commy2 is changing it, he is currently going through entire cba_common. But sure.